src/main/java/io/kestra/plugin/dbt/cli/Setup.java

```diff
@@ -32,6 +34,32 @@
     title = "Setup dbt",
     description = "Install pip package & init profile directory"
 )
+@Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Invoke dbt `build` command",
```

Comment:
Looks incorrect as we already have a Build task ? :thinking:

Reply:
Yes :(  I double-checked all those copy/paste, but one has slipped through.